docs/admin/config/site_config.mdx

```diff
@@ -220,7 +220,7 @@ All site configuration options and their default values are shown below.
 	//     "notifier": {
 	//       "channel": "#alerts",
 	//       "type": "slack",
-	//       "url": "https://hooks.slack.com/services/..."
+	//       "url": "https://hooks.slack.com/s/..."
```

Comment:
It would seem that Slack has updated their hooks URL? 

If that is what we're trying to fix it would be helpful to have a reference for this change (reply to this comment with it).